docs/src/manual/physics.md

```diff
@@ -0,0 +1,344 @@
+density distribution are
+```math
+    \partial_t \bm{u} + \left ( \bm{u} \bm{\cdot} \bm{\nabla} \right ) \bm{u} 
+        + \bm{f} \times \bm{u} = - \bm{\nabla} \phi + b \bm{\hat z} 
+        - \bm{\nabla} \bm{\cdot} \bm{\tau} + \bm{F_u} \, ,
+    \tag{eq:momentum}
+```
+where $\bm{\tau}$ is the kinematic stress tensor, $\bm{F_u}$
+denotes an internal forcing of the velocity field $\bm{u}$, $\phi$ is the potential 
+associated with kinematic and constant hydrostatic contributions to pressure, and 
+$\bm{f}$ is *Coriolis parameter*, or the background vorticity associated with the 
+specified rate of rotation of the frame of reference.
+
+## The tracer conservation equation
+
+The conservation law for tracers in `Oceananigans.jl` is
+```math
+    \partial_t c + \bm{u} \bm{\cdot} \bm{\nabla} c = - \bm{\nabla} \bm{\cdot} \bm{q}_c + F_c \, ,
+    \tag{eq:tracer}
+```
+where $\bm{q}_c$ is the diffusive flux of $c$ and $F_c$ is an arbitrary source term.
+`Oceananigans.jl` permits arbitrary tracers and thus an arbitrary number of tracer 
+equations to be solved simultaneously with the momentum equations.
+
+## Buoyancy model and equations of state
+
+The buoyancy model determines the relationship between tracers and the buoyancy $b$ in the momentum equation.
+
+### Buoyancy tracer
+
+The simplest buoyancy model uses buoyancy $b$ itself as a tracer: $b$ obeys the tracer
+conservation equation and is used directly in the momentum equations in the momentum equation.
+
+### Seawater buoyancy
+
+For seawater buoyancy is, in general, modeled as a function of conservative temperature 
+$\theta$, absolute salinity $S$, and depth below the ocean surface $d$ via
+```math
+    b = - \frac{g}{\rho_0} \rho' \left (\theta, S, d \right ) \, ,
+    \tag{eq:seawater-buoyancy}
+```
+where $g$ is gravitational acceleration, $\rho_0$ is the reference density.
+The function $\rho'(\theta, S, d)$ in the seawater buoyancy relationship that links conservative temperature, 
+salinity, and depth to the density perturbation is called the *equation of state*.
+Both $\theta$ and $S$ obey the tracer conservation equation.
+
+#### Linear equation of state
+
+Buoyancy is determined from a linear equation of state via
+```math
+    b = g \left ( \alpha_\theta \theta - \beta_S S \right ) \, ,
+```
+where $g$ is gravitational acceleration, $\alpha_\theta$ is the thermal expansion coefficient, 
+and $\beta_S$ is the haline contraction coefficient.
+
+#### Nonlinear equation of state 
+
+Buoyancy is determined by the simplified equations of state introduced by Roquet et al (2015).
+
+## Coriolis forces
+
+The Coriolis model controls the manifestation of the term $\bm{f} \times \bm{u}$ in the momentum equation.
+
+### The "$f$-plane" approximation
+
+Under an $f$-plane approximation[^3] the reference frame in which 
+the momentum and tracer equations are are solved rotates at a constant rate around a 
+vertical axis, such that 
+```math
+    \bm{f} \equiv f \bm{\hat z}
```

Comment:
Why use `=` for the beta-plane definition but `\equiv` for the f-plane?

Reply:
Its a typo